server/base/src/main/java/org/apache/accumulo/server/ServerInfo.java

```diff
@@ -111,10 +111,6 @@ public String getInstanceID() {
     return instanceID;
   }
 
-  public String getZooKeeperRoot() {
-    return zooKeeperRoot;
```

Comment:
This isn't used but might be nice to keep.  It could also be added again so I don't have strong feelings

Reply:
I like to remove stuff like this because it has a cascading effect, allowing removal of more.